test/lib/tasks/seed_rake_test.rb

```diff
@@ -21,7 +21,7 @@ def self.glob_dummy_seeds
     it 'creates all the seed tasks' do
       seeds = %w[db:seed:circular1 db:seed:circular2 db:seed:common db:seed:dependency db:seed:dependency2
                  db:seed:dependent db:seed:dependent_on_nested db:seed:dependent_on_several db:seed:development
-                 db:seed:development:users db:seed:no_block db:seed:original db:seed:reference_memos db:seed:with_block_memo db:seed:with_inline_memo]
+                 db:seed:development:users db:seed:no_block db:seed:original db:seed:reference_memos db:seed:replant db:seed:with_block_memo db:seed:with_inline_memo]
```

Comment:
`db:seed:replant` is added in rails 6 ([link](https://github.com/rails/rails/issues/34765))
I think it's better to bump rails too.